scripts/RAG.py

```diff
@@ -48,4 +49,26 @@ def generate_answer(self, query_text, k=4):
         except Exception as e:
             logging.error(f"Error generating answer: {e}")
             return "Error generating answer."
+    
+    def stream_answer(self, query_text, k=4):
+        """Generate an answer using the vector database and Ollama model with streaming output."""
```

Comment:
in the docstring please put description of the variables, query_text has an informative name, but not k.